README.md

```diff
@@ -313,6 +313,12 @@ You will need to run the backend with the `e2e` profile, and with the following
 OKTA_TESTING_DISABLEHTTPSCHECK=true ./gradlew bootRun --args='--spring.profiles.active=e2e'
 ```
 
+You will need to run the frontend locally:
```

Comment:
Can we roll these steps into a yarn task? This is becoming a very manual process.

Reply:
The process is the same, but the docs have more info to ensure everybody can run e2e tests locally.

I'll revisit the yarn task to remove some of these manual steps.

Reply:
I've rolled most steps into the yarn commands.